Core/Core/Use Cases/UploadManager.swift

```diff
@@ -239,7 +241,8 @@ open class UploadManager: NSObject, URLSessionDelegate, URLSessionTaskDelegate,
         Logger.shared.log()
         let objectID = file.objectID
         context.performAndWait {
-            guard let file = try? context.existingObject(with: objectID) as? File, let taskID = file.taskID else { return }
+            guard let file = try? context.existingObject(with: objectID) as? File else { return }
+            let taskID = file.taskID
```

Comment:
This change lets you `cancel` (ie delete from the uploader context) a file that isn't in progress (like one that had an error).